src/utils/presenter.ts

```diff
@@ -13,6 +15,14 @@ export const getPresenterPage = (req: Request, res: Response, next: NextFunction
     const appData: ApplicationData = getApplicationData(req.session);
     const presenter = appData[PresenterKey];
 
+    if (isRemoveJourney(req)){
+      return res.render(config.UPDATE_PRESENTER_PAGE, {
+        journey: config.JourneyType.remove,
+        backLinkUrl: `${config.UPDATE_FILING_DATE_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
+        templateName: config.UPDATE_PRESENTER_PAGE
+      });
```

Comment:
Don't you need the `...presenter` here too? A bit like in the call to `.render` on line 26?

Reply:
Now included. Looks good.